README.md

```diff
@@ -91,26 +91,39 @@ mvn spring-boot:stop
 # Make sure you are in the root of the repo
 # Run the command below to stop the previous instance and free up port 4120:
 
-mvn spring-boot:stop 
+mvn spring-boot:stop
 
 # You can also hit Ctrl-C from the shell window
 
-# Build the image and run the container using Docker 
+# Build the image and run the container using Docker
 
 # make sure you are in the root of the repo
 
 # this will use Azure CLI cached credentials
 
 # build the image
 
-docker build -t helium-dev .
+# IMPORTANT: In the following steps, you will be using the Dockerfile-Dev to build a developer
+# image, and thus, you MUST make sure that you change the UID of the 'helium' user set via the
+# 'useradd -u <uid>' in the Dockerfile-Dev to the one that is assigned to you on your host.
+# You may use the 'id' command to figure out your user id.
+
+docker build . -t helium-dev -f Dockerfile-Dev
 
-# Then run the container.  
+# Then run the container.
 
-docker run -p4120:4120 --name helium-dev --env AUTH_TYPE=CLI --env KEYVAULT_NAME=$KEYVAULT_NAME -v ~/.azure:/home/helium/.azure helium-dev:latest
+docker run -p4120:4120 --name helium-dev --env AUTH_TYPE=CLI --env KEYVAULT_NAME=$KEYVAULT_NAME -v ~/.azure:/home/helium/.azure -it helium-dev:latest
```

Comment:
since AUTH_TYPE is set in docker, it's redundant

Reply:
Where are the local environment variables set?  Helium setup uses $He_Name

Reply:
Since the refactor of the README, the He_Name has been moved over to the helium README and is not present in the individual language README files as one would need. Both the C# and Java README files do not contain references to He_Name. I checked typescript and it still contains references to He_Name for some reason. Would placing He_Name in the README be preferable?